tests/modules/programs/helix/languages-expected-use-grammars.toml

```diff
@@ -0,0 +1,12 @@
+[use-grammars]
+except = ["yaml", "json"]
+[[grammar]]
```

Comment:
i dont love that using `concatTextFile` gives us this very slightly ugly result without an additional break between sections, unsure if there is a workaround